config/src/main/java/com/typesafe/config/impl/Tokenizer.java

```diff
@@ -299,7 +299,7 @@ private Token pullComment(int firstChar) {
         // chars JSON allows to be part of a number
         static final String numberChars = "0123456789eE+-.";
         // chars that stop an unquoted string
-        static final String notInUnquotedText = "$\"{}[]:=,+#`^?!@*&\\";
+        static final String notInUnquotedText = "$\"{}()[]:=,+#`^?!@*&\\";
```

Comment:
I think they were allowed in unquoted strings before. Could add tests here for example https://github.com/typesafehub/config/blob/b076de21ceaa41789c80870609cb7435e09f2356/config/src/test/scala/com/typesafe/config/impl/TestUtils.scala#L423 and also in the tokenizer test if they are now parsed as tokens.